compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/platform/Actuals.jsNative.kt

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+@file:Suppress("ACTUAL_ANNOTATIONS_NOT_MATCH_EXPECT")
```

Comment:
IDE says it's a redundant suppression. 

The expect is:
```
internal expect fun Any.nativeClass(): Any
```

Looks like the actual matches the expect. Why was the suppress needed?

Reply:
It's from https://github.com/JetBrains/compose-multiplatform-core/commit/6afa1f98324a5dbc909c84f422c47f2e94c0601f based on git blame. It looks like git considered this as part of file rename at some point